Sources/Kitura/staticFileServer/StaticFileServer.swift

```diff
@@ -113,7 +115,10 @@ open class StaticFileServer: RouterMiddleware {
         #else
         let isDirectoryBool = isDirectory
         #endif
-        guard pathExists, isDirectoryBool else {
+        if !pathExists {
```

Comment:
:+1: Failable initializer or not, I prefer this approach, because users may have a legitimate reason for creating a server on a non-existent path (which may come into existence during the server's lifespan).